src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/clients/IGDBClientTests.kt

```diff
@@ -0,0 +1,177 @@
+package com.gamingbacklog.api.gamingbacklogapi.unit.clients
+
+import com.gamingbacklog.api.gamingbacklogapi.clients.IGDBClient
+import com.gamingbacklog.api.gamingbacklogapi.clients.util.ExternalAPIClient
+import com.gamingbacklog.api.gamingbacklogapi.unit.testutil.models.MockResponse
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.given
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.TestPropertySource
+
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+
+@SpringBootTest
+@Import(IGDBClient::class)
+@TestPropertySource(properties = ["CLIENT_ID=test_id", "CLIENT_SECRET=test_secret"])
+class IGDBClientTests {
+  lateinit var externalAPIClient: ExternalAPIClient
+  lateinit var igdbClient: IGDBClient
```

Comment:
Repeated from last review:
What's the benefit of using lateinit over instantiating it in the beginning?

Reply:
none as it turns out! It's a stylistic thing. I removed this and just declared on top since the beforeEach was therefore unnecessary